pkg/archive/copy.go

```diff
@@ -67,19 +67,28 @@ func SpecifiesCurrentDir(path string) bool {
 // SplitPathDirEntry splits the given path between its
 // parent directory and its basename in that directory.
 func SplitPathDirEntry(localizedPath string) (dir, base string) {
-	normalizedPath := filepath.ToSlash(localizedPath)
+	var normalizedPath string
+	// On Windows, ToSlash would convert the path to a Linux-style path which
+	// wouldn't work with filepath.* operations. Although it should already
+	// contain \, there is no harm ensuring this by using FromSlash instead.
+	if runtime.GOOS != "windows" {
```

Comment:
@jhowardmsft i'm sorry i must be tired but my brain just doesn't get it, do you have an example of the problem this is trying to solve?